hub/policy/src/integration-test/java/uk/gov/ida/integrationtest/hub/policy/StateControllerFactoryTest.java

```diff
@@ -220,6 +229,20 @@ public void build_shouldCreateCycle3DataInputCancelledController() {
         assertThat(controller).isInstanceOf(Cycle3DataInputCancelledStateController.class);
     }
 
+    @Test
+    public void build_shouldCreateUserAccountCreationFailedStateController() {
+        StateController controller = factory.build(aUserAccountCreationFailedState().build(), stateTransitionAction);
+
+        assertThat(controller).isInstanceOf(UserAccountCreationFailedStateController.class);
+    }
+
+    @Test
+    public void ShouldCreateCountryUserAccountCreationFailedStateControllerOnBuild() {
```

Comment:
Lower case method name please, Codacy is going to complain again.

Reply:
Typo! Typo!
Fixed :)